src/components/20-molecules/dropdown/index.js

```diff
@@ -369,6 +347,14 @@ class AXADropdown extends NoShadowDOM {
   updated() {
     this.updateTitle();
   }
+
+  disconnectedCallback() {
+    this.removeEventListener('focus', this.onFocus, false);
+    this.removeEventListener('blur', this.onBlur, false);
+    window.removeEventListener('resize', this.handleResize);
```

Comment:
dtto with window. better maybe `(window || global)`

Reply:
This does not make sense here, since most of the 10 instances of `window` relate to DOM API functions, and in node unit testing require `jsdom` anyway, which provides a `window` object.

Not done.